README.md

```diff
@@ -28,6 +28,26 @@ This repository uses [lerna](https://github.com/lerna/lerna) to manage it's pack
 | `react-shopify-app-route-propagator` | [README](packages/react-shopify-app-route-propagator/README.md)                          | [![npm version](https://badge.fury.io/js/%40shopify%2Freact-shopify-app-route-propagator.svg)](https://badge.fury.io/js/%40shopify%react-shopify-app-route-propagator) |
 | `react-shortcuts`                    | [README](packages/react-shortcuts/README.md)                                             | [![npm version](https://badge.fury.io/js/%40shopify%2Freact-shortcuts.svg)](https://badge.fury.io/js/%40shopify%2Freact-html)                                          |
 
+## Development
```

Comment:
To me this title sounds a bit like it would be tips on how to actually work on the repo

Reply:
should this be part of the `contributing` documentation?